packages/expo-cli/package.json

```
@@ -64,7 +64,7 @@
   "dependencies": {
     "@expo/build-tools": "^0.1.10",
     "@expo/bunyan": "3.0.2",
-    "@expo/config": "3.2.15",
+    "@expo/config": "^3.2.15",
```

```suggestion
    "@expo/config": "3.2.15",
```
Why this change? We're more or less trying to keep strict versions in package.json.